doc/releases/reef.rst

```diff
@@ -0,0 +1,219 @@
+====
+Reef
+====
+
+Reef is the 18th stable release of Ceph. It is named after the reef squid
+(sepioteuthis).
+
+v18.2.0 Reef
+============
+
+This is the first stable release of Ceph Reef.
+
+CephFS
+~~~~~~
+
+* CephFS: The ``mds_max_retries_on_remount_failure`` option has been renamed to
+  ``client_max_retries_on_remount_failure`` and moved from ``mds.yaml.in`` to
+  ``mds-client.yaml.in``. This change was made because the option has always
+  been used only by the MDS client.
+* CephFS: It is now possible to delete the recovered files in the
+  ``lost+found`` directory after a CephFS post has been recovered in accordance
+  with disaster recovery procedures.
+
+MGR
+~~~
+
+* mgr/snap_schedule: The snap-schedule mgr module now retains one snapshot less
+  than the number mentioned against the config tunable
+  ``mds_max_snaps_per_dir``. This means that a new snapshot can be created and
+  retained during the next schedule run.
+* The ``ceph mgr dump`` command now outputs ``last_failure_osd_epoch`` and
+  ``active_clients`` fields at the top level. Previously, these fields were
+  output under the ``always_on_modules`` field.
+  
+RADOS
+~~~~~
+
+* The trimming of PGLog dups is now controlled by the size instead of the
+  version. This fixes the PGLog inflation issue that occurred when the on-line
+  (in OSD) trimming got jammed after a PG-split operation. A new off-line
+  mechanism has been added: the ``ceph-objectstore-tool`` has added a
+  ``trim-pg-log-dups`` operation that targets situations where an OSD is unable
+  to boot due to inflated dups. When this is the case, the ``You can be hit by
+  THE DUPS BUG`` warning will be visible in the OSD logs.  Relevant tracker:
+  https://tracker.ceph.com/issues/53729
+* The RADOS python binding is now able to process (opt-in) omap keys as bytes
+  objects. This enables interacting with RADOS omap keys that are not
+  decodeable as UTF-8 strings.
+* mClock Scheduler: The mClock scheduler (the default scheduler in Quincy) has
+  undergone significant usability and design improvements to address the slow
+  backfill issue. The following is a list of some important changes:
+
+  * The ``balanced`` profile is set as the default mClock profile because it
+    represents a compromise between prioritizing client I/O and prioritizing
+    recovery I/O. Users can then choose either the ``high_client_ops`` profile
+    to prioritize client I/O or the ``high_recovery_ops`` profile to prioritize
+    recovery I/O.
+  * QoS parameters like ``reservation`` and ``limit`` are now specified in 
+    terms of a percentage (range: 0.0 to 1.0) of the OSD's IOPS capacity.
```

Comment:
s/percentage/fraction/

Reply:
But this isn't a fraction. It's a decimal value.

Reply:
```
frac·tion
noun
noun: fraction; plural noun: fractions; noun: Fraction; noun: the Fraction

    1.
    a numerical quantity that is not a whole number (e.g. 1/2, 0.5).
    2.
    a small or tiny part, amount, or proportion of something.
    "he hesitated for a fraction of a second"```

Reply:
Historically in Ceph we have had some quantities expressed as fractions, e.g. [0-100] in output or configuration, and others within the range [0-1], e.g. OSD override reweights.  At times I've seen people be confused as to the appropriate / allowable range for a given configuration so my goal here was to align with clarity.

Reply:
Okay. I'm convinced.